SUI/SUI/View/GreetingView.swift

```diff
@@ -0,0 +1,99 @@
+import SwiftUI
+
+struct GreetingView: View {
+    private struct Constants {
+        static let siteName = "169.ru"
+        static let getStartedButtonText = "Get Started"
+        static let noAccountText = "Don't have an account?"
+        static let signInText = "Sign in here"
+        static let imageUrlStrings = [
+            "https://cdni.iconscout.com/illustration/premium/thumb/living-room-interior-7209100-5870046.png",
+            "https://cdni.iconscout.com/illustration/premium/thumb/modern-living-room-3736054-3123364.png",
+            "https://cdni.iconscout.com/illustration/premium/thumb/bedroom-interior-7209115-5870061.png?f=webp"
+        ]
+    }
+
+    var body: some View {
+        VStack {
+            Header
+            Spacer()
+
+            Button(Constants.getStartedButtonText) {
+                openProductDetailsView()
+            }
+            .buttonStyle(.white)
+            .fullScreenCover(isPresented: $isProductDetailsFlowPreseneted, content: {
+                ProductDetailView()
+            })
+
+            Spacer()
+                .frame(maxHeight: 75)
+
+            SignInFooter
+        }
+        .padding(.horizontal, 47)
+        .padding(.vertical, 60)
+        .foregroundStyle(.white)
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .background {
+            LinearGradient.backgroundVertical
+                .ignoresSafeArea()
+        }
+    }
+
+    private var Header: some View {
```

Comment:
```suggestion
    private var headerView: some View {
```
Все же это не тип, а просто экземпляр. К каждой такой переменной добавляем View в конце